tests/Hakyll/Web/Pandoc/Biblio/Tests.hs

```diff
@@ -64,3 +65,36 @@ goldenTest01 =
             cleanTestEnv
 
             return output)
+
+goldenTest02 :: TestTree
+goldenTest02 =
+    goldenVsString
+        "biblio02"
+        (goldenTestsDataDir </> "biblio01.golden")
```

Comment:
Since this file is not used in multiple tests, please rename it to something like `cites-meijer.golden" or something. `Extra-source-files` in hakyll.cabal will have to be updated accordingly.